tests/integration/base.py

```diff
@@ -605,6 +605,12 @@ def _assertTablesEqualSql(self, relation_a, relation_b, columns=None):
 
         return sql
 
+    def assert_in_log(self, message) -> None:
```

Comment:
The only other place this behavior exists today checks for several different messages.  For the most part you shouldn't look at the log, since you can just as easily look at stdout with run_and_capture; here I specifically need to look at debug messages, which I'm only doing because I don't know another good way to observe optimize.  So short answer is, I may expand this in the future to take multiple messages, at which point that other test can fold in, but I also don't want to encourage log scanning lol.